examples/simple-platform/user-walk-through.md

```diff
@@ -0,0 +1,48 @@
+# User Walk-through of the Platform Example
+
+## Creating an account and going through the GitHub OAuth loop
+
+Open the accounts app in your browser at `http://localhost:3025`. If you open 
+your browser's developer tools to the Console tab, you can see some 
+`interbit-middleware` commands being printed. The accounts app is fully up and 
+running once you see the `interbit-middleware/CHAIN_BLOCKING` status for the 
+`accountsPrivate` chain. This may take up to a minute. 
+
+From the main accounts app page, click on the "Create Account / Sign in" link 
+in the header, which navigates you to the `create-account` page. Click the 
+"Create Account" button, check that you have read the disclaimer and click 
+"Continue" in the modal.
+
+If you're already signed into your GitHub account, you will be directed to the 
+your personal account page. Otherwise, you will be redirected to GitHub and 
+prompted to enter your GitHub credentials. Enter your credentials and sign in. 
+You should be redirected back to the accounts app. 
+
+You will see a number of chain actions being dispatched in the console. 
```

Comment:
Which console? It is generally helpful to monitor the `platform-deploy` console to see actions that are dispatched to the account and template chains.